gogoproto/helper.go

```diff
@@ -351,3 +351,7 @@ func ImportsGoGoProto(file *google_protobuf.FileDescriptorProto) bool {
 func HasCompare(file *google_protobuf.FileDescriptorProto, message *google_protobuf.DescriptorProto) bool {
 	return proto.GetBoolExtension(message.Options, E_Compare, proto.GetBoolExtension(file.Options, E_CompareAll, false))
 }
+
+func EnabledDualRegistration(file *google_protobuf.FileDescriptorProto) bool {
```

Comment:
How about calling this RegistersGolangProto or something like and not say anything about Dual